application/server/server.go

```diff
@@ -276,8 +278,19 @@ func initializeHandler(srv *jrpc2.Server, c *config.Config) handler.Func {
 }
 func initializedHandler(srv *jrpc2.Server) handler.Func {
 	return handler.New(func(ctx context.Context, params lsp.InitializedParams) (any, error) {
+		// Logging these messages only after the client has been initialized.
+		// It has been observed that VSCode changes the log messages format after the LS has been initialized.
+		// No reason to log the method name for these messages, because some of these values are empty and the messages
+		// looks weird when including the method name.
+		c := config.CurrentConfig()
+		log.Info().Msg("snyk-ls: " + config.Version + " (" + util.Result(os.Executable()) + ")")
+		log.Info().Msg("platform: " + runtime.GOOS + "/" + runtime.GOARCH)
+		log.Info().Msg("https_proxy: " + os.Getenv("HTTPS_PROXY"))
+		log.Info().Msg("http_proxy: " + os.Getenv("HTTP_PROXY"))
+		log.Info().Msg("no_proxy: " + os.Getenv("NO_PROXY"))
+		log.Info().Msg("IDE: " + c.IntegrationName() + "/" + c.IdeVersion())
```

Comment:
Is this correct? Is integration name the IDE in all cases?

Reply:
Maybe clientInfo.name instead? `"clientInfo":{"name":"Sublime Text LSP","version":"1.24.0"}`

Reply:
For the mainstream plugins, it's going to be the same.
For unofficial IDE, `setClientInformation` is going to set integration name to the ClientInfo.Name, so it's going to be the same there as well